src/couch_mrview_http.erl

```diff
@@ -433,6 +443,7 @@ parse_params(Props, Keys) ->
 
 parse_params(Props, Keys, #mrargs{}=Args0) ->
     Args = Args0#mrargs{keys=Keys},
+    [check_compatible(K1, K2, Props) || {K1, K2} <- [{"group", "group_level"}]],
```

Comment:
yeah, I would rather see the the validation happen further down, compare with how we parse the 'stale' param, we match the allowed values, and throw for anything else. In this case, we'd throw in the <<"group">> and <<"group_level">> clause if Args#mrargs.group_level /= undefined.